packages/lit-dev-content/site/tutorials/content/carousel/08.md

```diff
@@ -0,0 +1,45 @@
+Now we're ready to add the animation. To do so, we'll import the `animate`
+directive from the `@lit-labs/motion` package.
+
+<aside class="warning">
+  Packages release under the <code>@lit-labs</code> umbrella aren't quite ready for
+  production. These projects may be experimental or incomplete, and they're
+  subject to breaking changes. See the <a href="https://lit.dev/docs/libraries/labs/">docs</a>
+  for more info.
+</aside>
```

Comment:
This can be a `<litdev-aside type="warn" no-header>`. See: https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/samples/tutorials/CONTRIBUTING.md#asides

The content in the aside can also be markdown (provided there is one line of whitespace between the tag and the content).

Also prefer link without the domain, i.e.: `/docs/libraries/labs/`